winforms/src/toga_winforms/widgets/textinput.py

```diff
@@ -120,6 +102,3 @@ def clear_error(self):
 
     def set_error(self, error_message):
         self.error_provider.SetError(self.native, error_message)
-
-    def winforms_double_click(self, sender, event):
-        self.native.SelectAll()
```

Comment:
I don't know why this was here: double-clicking on Windows is supposed to select one word, not the entire box.

Reply:
Yeah - I can't explain this one either.